src/test/java/org/folio/rest/HeadersForwardingTest.java

```diff
@@ -209,5 +225,15 @@ public void testPostBlUsersPasswordResetReset() {
         .withHeader(HttpHeaders.USER_AGENT, WireMock.matching(".+"))
         .withHeader(BLUsersAPI.X_FORWARDED_FOR_HEADER, WireMock.equalTo(IP))
     );
+
+    WireMock.getAllServeEvents().stream()
+      .map(ServeEvent::getRequest)
+      .forEach(this::verifyHeaders);
+  }
+
+  private void verifyHeaders(LoggedRequest request) {
+    com.github.tomakehurst.wiremock.http.HttpHeaders headers = request.getHeaders();
```

Comment:
please replace to `import javax.ws.rs.core.HttpHeaders` by  `import static org.apache.http.HttpHeaders.USER_AGENT` and then you can import `com.github.tomakehurst.wiremock.http.HttpHeaders` instead of using full class name

Reply:
Done